c2corg_ui/views/document.py

```diff
@@ -68,13 +74,34 @@ def _get_document(self, id, culture):
         return document, locale
 
     def _get_documents(self):
-        url = '%s/%s' % (self.settings['api_url'], self._API_ROUTE)
+        params = self._get_filter_params()
+        query_string = '?' + urlencode(params) if params else ''
+        url = '%s/%s%s' % (
+            self.settings['api_url'], self._API_ROUTE, query_string
+        )
         resp, content = self._call_api(url)
+        filters = dict(self._DEFAULT_FILTERS, **{k: v for k, v in params})
         # TODO: better error handling
         if resp['status'] == '200':
-            return content['documents'], content['total']
+            documents = content['documents']
+            total = content['total']
         else:
-            return [], 0
+            documents = []
+            total = 0
+        return documents, total, filters
+
+    def _get_filter_params(self):
+        params = []
+        if 'filters' in self.request.matchdict:
+            filters = self.request.matchdict['filters']
+            # If number of filters is odd, add an empty string at the end:
```

Comment:
I though we might have the case of filter params without value (a bit like ``?debug``). I don't remember if this is already possible in v5.
Maybe you're right and this is not useful.